protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -224,12 +225,89 @@ async fn test_eth_client_should_successfully_call_initiate_transfer() {
 			RecipientAddress(recipient),
 			HashLock(hash_lock),
 			TimeLock(100),
-			Amount(1000), // Eth
+			// value has to be > 0
+			Amount(AssetType::EthAndWeth((1, 0))), // Eth
 		)
 		.await
 		.expect("Failed to initiate bridge transfer");
 }
 
+#[tokio::test]
+async fn test_eth_client_should_successfully_call_initiate_transfer_only_weth() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	let signer_address = harness.set_eth_signer(anvil.keys()[0].clone());
+
+	harness.deploy_init_contracts().await;
+
+	let recipient = harness.gen_aptos_account();
+	let hash_lock: [u8; 32] = keccak256("secret".to_string().as_bytes()).into();
+	harness
+		.initiate_bridge_transfer(
+			InitiatorAddress(EthAddress(signer_address)),
+			RecipientAddress(recipient),
+			HashLock(hash_lock),
+			TimeLock(100),
+			Amount(AssetType::EthAndWeth((0, 1))),
+		)
+		.await
+		.expect("Failed to initiate bridge transfer");
+}
+
+#[tokio::test]
+async fn test_eth_client_should_successfully_call_initiate_transfer_eth_and_weth() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	let signer_address = harness.set_eth_signer(anvil.keys()[0].clone());
+	let matching_signer_address = harness.eth_signer_address();
+
+	assert_eq!(signer_address, matching_signer_address, "Signer address mismatch");
+
+	harness.deploy_init_contracts().await;
+
+	let recipient = harness.gen_aptos_account();
+	let hash_lock: [u8; 32] = keccak256("secret".to_string().as_bytes()).into();
+	harness
+		.initiate_bridge_transfer(
+			InitiatorAddress(EthAddress(signer_address)),
+			RecipientAddress(recipient),
+			HashLock(hash_lock),
+			TimeLock(100),
+			Amount(AssetType::EthAndWeth((1, 1))),
+		)
+		.await
+		.expect("Failed to initiate bridge transfer");
+}
```

Comment:
We're missing the case `test_eth_client_should_successfully_call_initiate_transfer_only_eth`, let's add it.

Reply:
it's above, it's not named _only_eth